gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,6 +66,11 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def raise_failover_event
+      require "awesome_spawn"
+      AwesomeSpawn.run("rake evm:db:raise_failover_executed_event")
```

Comment:
If the rake task becomes generic, I would move the definition from `evm_dba.rake` to `evm.rake` also.